resources/qml/UserSettingsPage.qml

```diff
@@ -0,0 +1,218 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+// SPDX-FileCopyrightText: 2022 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import "ui"
+import Qt.labs.platform 1.1 as Platform
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.13
+import im.nheko 1.0
+
+Rectangle {
+    id: userSettingsDialog
+
+    property bool collapsed: width < 800
+    color: Nheko.colors.window
+
+    Shortcut {
+        sequence: StandardKey.Cancel
```

Comment:
What key is `StandardKey.Cancel` supposed to map to?

If it's supposed to be 'escape' that doesn't do anything on my end.  I also tried 'delete' and 'backspace' to the same end.

Reply:
Escape. Same as on the Roomlist. Buuuuuuut, that code was supposed to be gone anyway, it is a left over from me having both side by side, the old and the new page and comparing them.